src/lib/viewers/media/DashViewer.js

```diff
@@ -166,7 +166,12 @@ class DashViewer extends VideoBaseViewer {
             }
         });
         this.player.getNetworkingEngine().registerRequestFilter(this.requestFilter);
-        this.player.load(this.mediaUrl);
+        return this.player.load(this.mediaUrl).catch((error) => {
```

Comment:
Is that not caught by this `this.player.addEventListener('error', this.shakaErrorHandler);` on line 152?

Reply:
@jeremypress no it doesnt appear to be. If you force an error like a 404, it throws an uncaught exception

Reply:
Since we've added the new error metrics stuff, can you see if this is causing a "preview_error" event to be emitted, by Preview? Also, would you please change `shakeErrorHandler()` to use `this.triggerError(error)` instead of `this.emit('error', error)`?

Thanks!